Sources/Apodini/Response/Response.swift

```diff
@@ -18,32 +18,33 @@ public struct Response<Content: Encodable>: ResponseTransformable {
         Response<Content>(connectionEffect: .close)
     }
     
-    
-    public static func send(_ content: Self.Content, status: Status? = nil) -> Response<Self.Content> {
-        Response<Self.Content>(status: status, content: content, connectionEffect: .open)
+    public static func send(_ content: Self.Content, status: Status? = nil, information: Set<AnyInformation> = []) -> Response<Content> {
```

Comment:
You could remove the Self from Self.Content here as well to stay consistent with the other Content usages. Same for line 29